Index: co-main/src/main/java/com/cccs7/MainApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.cccs7;\r\n\r\n/**\r\n * <p> $END </p>\r\n *\r\n * @Author cccs7/cs7eric - csq020611@gmail.com\r\n * @Description $END\r\n * @Date 2023/9/8 12:18\r\n */\r\npublic class Main {\r\n    public static void main(String[] args) {\r\n        System.out.println(\"Hello world!\");\r\n    }\r\n}
===================================================================
diff --git a/co-main/src/main/java/com/cccs7/MainApplication.java b/co-main/src/main/java/com/cccs7/MainApplication.java
--- a/co-main/src/main/java/com/cccs7/MainApplication.java	
+++ b/co-main/src/main/java/com/cccs7/MainApplication.java	
@@ -1,14 +1,22 @@
 package com.cccs7;
 
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
 /**
- * <p> $END </p>
+ * <p> MainApplication 启动类 </p>
  *
  * @Author cccs7/cs7eric - csq020611@gmail.com
  * @Description $END
  * @Date 2023/9/8 12:18
  */
-public class Main {
+@EnableWebMvc
+@SpringBootApplication
+public class MainApplication {
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        SpringApplication.run(MainApplication.class);
     }
 }
\ No newline at end of file
